From 464a207f55743d289980b3196470515d9234e44d Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Mon, 25 Jul 2022 11:42:29 +0300
Subject: [PATCH] [SW-86607] gaudi3: add fw sram params

FW SRAM parameters are added, it will be used in the next
patch for the SPI DMA parameters generation.

Change-Id: I726a0005a73590bfc3fc678cca37bfcb157415be
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index 6d3e48d..50ec748 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -15,6 +15,8 @@
 
 #define HL_SP_SRAM_BASE			0x300007FD3120000ULL
 #define HL_SP_SRAM_SIZE			SZ_64K
+#define HL_FW_SRAM_FULL_BASE		0x300007FD3140000ULL
+#define HL_FW_SRAM_SIZE			SZ_128K
 
 #define ARC0_MEM_SIZE			(ARC0_ICCM_SIZE + ARC0_DCCM_SIZE)
 #define ARC1_MEM_SIZE			(ARC1_ICCM_SIZE + ARC1_DCCM_SIZE)
